aaelf64-morello/aaelf64-morello.rst

```diff
@@ -0,0 +1,825 @@
+..
+   Copyright (c) 2011-2020, Arm Limited and its affiliates.  All rights reserved.
+   CC-BY-SA-4.0 AND Apache-Patent-License
+   See LICENSE file for details
+
+.. |release| replace:: 2020Q3
```

Comment:
to avoid confusion (there is no 2020q3 release based on these sources), I'd change this to the next release: 2020Q4